packages/node/src/models/proposed-app-instance-info.ts

```diff
@@ -50,7 +49,7 @@ export interface AppInstanceProposalJSON {
  * This class captures said state for the duration of the proposal being made and
  * the respecting `AppInstance` is installed.
  */
-export class AppInstanceProposal implements AppInstanceInfo {
+export class AppInstanceProposal {
```

Comment:
Is this class needed now that proposed and installed `AppInstance`s share the same model?